lib/loggers/index.js

```diff
@@ -28,13 +29,13 @@ cc({
 
 // create custom logging levels
 const levels = {
-  silly: 0,
-  verbose: 1,
-  debug: 2,
+  silly: 6,
+  verbose: 5,
+  debug: 4,
   info: 3,
-  warning: 4,
-  error: 5,
-  critical: 6
+  warning: 2,
+  error: 1,
+  critical: 0
```

Comment:
I guess these all got reversed in later versions of winston.

Reply:
I don't know who did what when, but the ones we had were definitely in the wrong order.  See: https://github.com/winstonjs/winston#logging-levels